pkgs/checks/test/extensions/collection_equality_test.dart

```diff
@@ -51,30 +51,37 @@ void main() {
         'a',
         'b'
       ], [
-        it()
-          ..isA<String>().which(it()
-            ..startsWith('a')
-            ..length.isLessThan(2)),
-        it()..isA<String>().startsWith('b')
+        (Subject<dynamic> it) => it.isA<String>().which((it) => it
```

Comment:
If you forget to put the argument type, then `it` is `dynamic`, and there is no static warning.

At runtime, the `.isA` call is made on the `Subject` instance because it was not resolved statically to the extension method.

Reply:
That does seem pretty annoying, but I will leave the decision up to you.